src/components/events-list/event-type.js

```diff
@@ -5,15 +5,11 @@ import { StatusBadge } from '../status-badge';
 import eventModel from '../../models/event';
 import { isWarningEvent } from '../../utils/event-model';
 
-const EventType = ({ event }) => {
-  const type = isWarningEvent(event) ? 'warning' : 'success';
-
-  return (
-    <React.Fragment>
-      <StatusBadge type={type}>{event?.type}</StatusBadge>
-    </React.Fragment>
-  );
-};
+const EventType = ({ event }) => (
+  <StatusBadge type={isWarningEvent(event) ? 'warning' : 'success'}>
+    {event?.type}
+  </StatusBadge>
```

Comment:
we can probably delete this whole component if you agree it is excessive.

Reply:
This file was deleted in the refactor